app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -75,6 +83,26 @@ class SessionDetailFragment : Fragment(), Injectable {
         }
 
         binding.toolbar.setNavigationOnClickListener { activity?.finish() }
+
+        binding.speakerSummary.viewTreeObserver.addOnPreDrawListener(
```

Comment:
Can you add an extension method to ViewExt?

```
fun View. addOnetimeOnPreDrawListener(listener:()->Unit)
```

https://github.com/takahirom/material-element/blob/490090906716d129c26a5329930b5c41b9e84724/app/src/main/java/com/github/takahirom/materialelement/animation/OnetimeViewTreeObserver.java#L7